op-deployer/pkg/deployer/manage/dependencies_test.go

```diff
@@ -79,6 +79,7 @@ func TestDependencies(t *testing.T) {
 				BaseFeeVaultRecipient:      addrFor(devkeys.BaseFeeVaultRecipientRole),
 				L1FeeVaultRecipient:        addrFor(devkeys.L1FeeVaultRecipientRole),
 				SequencerFeeVaultRecipient: addrFor(devkeys.SequencerFeeVaultRecipientRole),
+				OperatorFeeVaultRecipient:  addrFor(devkeys.OperatorFeeVaultRecipientRole),
```

Comment:
should be able to delete per https://github.com/ethereum-optimism/optimism/pull/12166/files#r1941779240